tools/gcs-usage-recommender/python/backfill.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import json
+import sys
+import logging
+from typing import Dict, List
+from google.cloud import resource_manager
+from google.cloud import storage
+
+
+def get_storage_client() -> storage.Client:
```

Comment:
Yup, makes sense. I initially had some more GCS functionality in there, but ended up getting rid of it. But since it's only used once now, it's no longer needed. I can delete it.